### PR TITLE
Remove libaflfuzzer from core, enable seed shuffling for afl + variants

### PR DESCRIPTION
**Pull request by vanhauser-thc**
For #814

### CODE REVIEW

File: fuzzers/aflcc/fuzzer.py [hunk 1]
```diff
@@ -272,6 +272,7 @@ def run_fuzzer(input_corpus,
         # performs.
         '2147483647'
     ]
+    os.environ['AFL_SHUFFLE_QUEUE'] = '1'
```
Comment (inferno-chromium) [inline fuzzers/aflcc/fuzzer.py:275]: This wont be needed since it calls into afl's prepare_fuzz_environment.

**Review by inferno-chromium — CHANGES_REQUESTED** [review]
LGTM with minor change needed.

File: fuzzers/afl/fuzzer.py [hunk 2]
```diff
@@ -124,6 +124,7 @@ def run_afl_fuzz(input_corpus,
         # performs.
         '2147483647'
     ]
+    os.environ['AFL_SHUFFLE_QUEUE'] = '1'
```
Comment (inferno-chromium) [inline fuzzers/afl/fuzzer.py:127]: Please move this into prepare_fuzz_environment

File: fuzzers/aflcc/fuzzer.py (hunk 1 above)
Comment (vanhauser-thc) [reply fuzzers/aflcc/fuzzer.py:275]: ok

File: fuzzers/afl/fuzzer.py (hunk 2 above)
Comment (vanhauser-thc) [reply fuzzers/afl/fuzzer.py]: done

File: fuzzers/weizz_qemu/fuzzer.py [hunk 3]
```diff
@@ -42,6 +42,7 @@ def fuzz(input_corpus, output_corpus, target_binary):
     os.environ['WEIZZ_I_DONT_CARE_ABOUT_MISSING_CRASHES'] = '1'
     os.environ['WEIZZ_CTX_SENSITIVE'] = '1'
     os.environ['WEIZZ_SKIP_CRASHES'] = '1'
+    os.environ['AFL_SHUFFLE_QUEUE'] = '1'  # not sure this is supported
```
Comment (inferno-chromium) [inline fuzzers/weizz_qemu/fuzzer.py:45]: Does it use AFL_ prefix or WEIZZ_ ?

**Review by inferno-chromium — APPROVED** [review]
LGTM with a question.

File: fuzzers/weizz_qemu/fuzzer.py (hunk 3 above)
Comment (inferno-chromium) [reply fuzzers/weizz_qemu/fuzzer.py:45]: Yes it is WEIZZ_SHUFFLE_QUEUE  
can you make the change
https://github.com/andreafioraldi/weizz-fuzzer/blob/1a2121029c0ed23e1529830dc5f4d9acabdebb10/src/main.c#L556

**Review by inferno-chromium — CHANGES_REQUESTED** [review]
minor fix needed, see comment.

File: fuzzers/weizz_qemu/fuzzer.py (hunk 3 above)
Comment (vanhauser-thc) [reply fuzzers/weizz_qemu/fuzzer.py]: had to check the repo. it actually supports it, but only with WEIZZ_ ... changed that

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**